datadotworld/client/api.py

```diff
@@ -441,6 +445,314 @@ def download_datapackage(self, dataset_key, dest_dir):
+
+    def fetch_datasets(self):
+        """Fetch authenticated user owned datasets
+
+        Parameters
+        ----------
+        limit : str, optional
+            Maximum number of items to include in a page of results
+        next : str, optional
+            Token from previous result page (to be used when requesting a subsequent page)
+        sort : str, optional
+            Property name to sort
+
+        Returns
+        -------
+        dict
+            Dataset definition, with all attributes
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> user_owned_dataset = api_client.fetch_datasets()
+        >>> user_owned_dataset.records
+        []
+        """
+        try:
+            return self._user_api.fetch_datasets()
+        except _swagger.rest.ApiException as e:
+            raise RestApiError(cause=e)
+
+    # Sql Operations
+
+    def sql_get(self, dataset_key, query):
+        """Executes SQL queries against a dataset via GET
+
+        Parameters
+        ----------
+        dataset_key : str
+            Dataset identifier, in the form of owner/id
+        query : str
+            SQL query
+        include_table_schema : bool
+            Flags indicating to include table schema in the response
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> query = 'SELECT * FROM test-dataset'
+        >>> api_client.sql_get('username/test-dataset', query)
+        """
+        owner_id, dataset_id = parse_dataset_key(dataset_key)
+        try:
+            return self._sql_api.sql_get(owner_id, dataset_id, query)
+        except _swagger.rest.ApiException as e:
+            raise RestApiError(cause=e)
+
+    def sql_post(self, dataset_key, query):
+        """Executes SQL queries against a dataset via POST
+
+        Parameters
+        ----------
+        dataset_key : str
+            Dataset identifier, in the form of owner/id
+        query : str
+            SQL query
+        include_table_schema : bool
+            Flags indicating to include table schema in the response
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> api_client.sql_post('username/test-dataset', query)
+        """
+        owner_id, dataset_id = parse_dataset_key(dataset_key)
+        try:
+            return self._sql_api.sql_post(owner_id, dataset_id, query)
+        except _swagger.rest.ApiException as e:
+            raise RestApiError(cause=e)
+
+    # Sparql Operations
+
+    def sparql_get(self, dataset_key, query):
```

Comment:
There is no need to expose both `sparql_get` and `sparql_post` at this level. Please remove `sparql_get` and rename `sparql_post` to `sparql`.